src/Microsoft.Identity.Client/Internal/Broker/BrokerInteractiveRequest.cs

```diff
@@ -17,8 +17,8 @@ namespace Microsoft.Identity.Client.Internal.Broker
     internal class BrokerInteractiveRequest
     {
         internal Dictionary<string, string> _brokerPayload = new Dictionary<string, string>();
-        readonly BrokerFactory brokerFactory = new BrokerFactory();
-        private IBroker Broker;
+        readonly BrokerFactory _brokerFactory = new BrokerFactory();
```

Comment:
How is BrokerInteractiveRequest tested if you new up a BrokerFactory every time? I would expect the BrokerInteractiveRequest to receive a broker factory in the ctor OR the broker factory to be in PlatformProxy like our other factories. 

You'd want to test BrokerInteractiveRequest with using a TestBroker that is a mock of IBroker, and this needs to be injected somehow.